app/controllers/ops_controller/ops_rbac.rb

```diff
@@ -845,8 +859,9 @@ def process_roles(roles, task)
     process_elements(roles, MiqUserRole, task)
   end
 
-  def process_tenants(tenants, task)
-    process_elements(tenants, Tenant, task, "Tenant", "name")
+  def process_tenants(tenants, task, display_model = 'Tenant')
```

Comment:
just pass in `display_model` name for both (unless this is called in a number of places)

But why are you partitioning these?
they are all records in the same table. Can't we just delete them?